osu.Game/Overlays/Mods/ModSelectOverlay.cs

```diff
@@ -172,6 +172,8 @@ public ModSelectOverlay()
                                     AutoSizeAxes = Axes.Y,
                                     Spacing = new Vector2(0f, 10f),
                                     Width = content_width,
+                                    LayoutDuration = 200,
+                                    LayoutEasing = Easing.OutQuint,
```

Comment:
@peppy Because of FillFlowContainer's spacing
So i use Show/Hide instead
.
https://streamable.com/m654k

Reply:
see my above comment.

Reply:
oops i already fixed that but forgot to push :P

Reply:
Am I waiting on a further commit to re-review this?

Reply:
You can review it now : )